code/backend/batch/utilities/helpers/AzureSearchHelper.py

```diff
@@ -7,6 +8,21 @@
 )
 from .LLMHelper import LLMHelper
 from .EnvHelper import EnvHelper
+from azure.search.documents.indexes import SearchIndexClient
+from azure.search.documents import SearchClient
+from azure.core.credentials import AzureKeyCredential
+from azure.identity import DefaultAzureCredential
+from azure.search.documents.indexes.models import (
```

Comment:
nit: Haven't we got an import above that uses `azure.search.documents.indexes.models`